src/parsimmon.js

```diff
@@ -841,4 +1000,21 @@ Parsimmon.whitespace = whitespace;
 Parsimmon["fantasy-land/empty"] = empty;
 Parsimmon["fantasy-land/of"] = succeed;
 
+function ensureBuffer(f) {
+  return function() {
+    if (typeof Buffer === "undefined") {
+      throw new Error(
+        "Buffer global does not exist; please consider using https://github.com/feross/buffer if you are running Parsimmon in a browser."
+      );
+    }
+    return f.apply(null, arguments);
+  };
+}
```

Comment:
I would rather this was just a function you call inside each of those `Parsimmon.Binary` functions because this is going to mess up the function names inside the console and have them all show up like:

```js
> Parsimmon.Binary.bitSeq
[Function: ensureBuffer]
```

Which will look weird, I think.